pkg/networkmanager/firewall.jsx

```diff
@@ -536,7 +535,7 @@ class AddEditServicesModal extends React.Component {
 
             if (oldState.generate_custom_id) {
                 if (all_ports.length > 0)
-                    newState.custom_id = "custom--" + all_ports.map(this.getName).join('-');
+                    newState.custom_id = "custom--" + all_ports.map(port => this.getName(port, type)).join('-');
```

Comment:
The type variable here is undefined. This is not even compiling probably? Did you try to build/test your changes before pushing?

Reply:
> The type variable here is undefined. This is not even compiling probably? Did you try to build/test your changes before pushing?

While trying to reset the incorrectly commited files, I ended up closing this PR. To reopen the pull request, I had made a force commit. I have a few changes in the pipeline which when I commit, will resolve the errors. Also I am having a bit hard time trying to set up the local environment. Been trying that continuously for over a week now. I need some help in setting up cockpit so that I can test my changes properly.

Reply:
> 

Ok @KKoukiou. I will connect with you on IRC.🫡

Reply:
> The type variable here is undefined. This is not even compiling probably? Did you try to build/test your changes before pushing?

If the issue of "type" variable being undefined is fixed, will this be correct usage of getName( ) call here. What do you suggest @KKoukiou ?